docs/content/en/docs/Getting started/understanding-the-technologies/index.md

```diff
@@ -3,90 +3,368 @@ title: "0. Understanding the technologies"
+
+#### Architecting for scale
+
+{{< imgproc iiot_1.jpg Fit "1280x500" >}}Architecting for scale.{{< /imgproc >}}
+
+#### Best-practices 1: Avoid common traps in the IIoT space
+
+*See also [Open source in Industrial IoT: an open and robust infrastructure instead of reinventing the wheel](/docs/concepts/open-source-industrial-iot/)*
+
+##### Avoid lock-in effects
+- Use open and freely accessible standards
+- Use open-source whenever reasonable
+- When acquiring new software, hardware or machines define contracts on data property, security plans and service levels
+- Check for and request full and understandable documentation of each device and interface
+
+##### Use both of best worlds (IT and OT)
+
+Look into the other world to get alternative solutions and inspiration e.g.
+- Grafana dashboard instead of a built in HMI
+- Industrial PCs instead of Raspberry PIs
+- Secure crucial connections with firewalls (e.g. pfSense)
+
+##### Avoid quick fixes
+- Use industry-wide IT/OT standards wherever applicable before developing on your own
+- Invest enough time for architecture basics and the big picture to avoid rework in the near future
+- Always document your project even when it seem unnecessary at the moment 
+
+#### Best-practices 2: Protocols which allow communication of IT and OT systems
+
+##### Coming from IT: MQTT
+{{< imgproc iiot_2.png Fit "1280x500" >}}MQTT{{< /imgproc >}}
+- A light weight with low-bandwidth and power requirements which is the leading standard for IoT applications 
+- All devices connect to a MQTT Broker which saves and distributes information to devices which subscribed to it (similar to social network)
+
+##### Coming from IT: REST API
+{{< imgproc iiot_3.png Fit "1280x500" >}}REST API{{< /imgproc >}}
+- Standard web application interface which is used on almost every website
+- Can be used to request information from the web e.g. weather data or distance between two cities (Google Maps) or request actions e.g. save a file in the database
+
+##### Coming from OT: OPC/UA
+{{< imgproc iiot_4.png Fit "1280x500" >}}OPC/UA{{< /imgproc >}}
+- Standard interface for automation and machine connectivity
+- Highly complex protocol with wide range of capabilities but low in user friendliness
+
+#### Best-practices 3: Reduce complexity in machine connection with tools like Node-RED
+
+*See also [Node-RED in Industrial IoT: a growing standard](/docs/concepts/node-red-in-industrial-iot/)*
+
+#### Best-practices 4: Connect IT and OT securely using a Demilitarized Zone (DMZ)
+
+*See also [Why are our networks open by default and how do I protect my valuable industrial assets?](https://www.linkedin.com/pulse/why-our-networks-open-default-how-do-i-protect-my-valuable-proch/?trk=articles_directory)*
+
+#### Architecture
+
+*See also [Architecture](/docs/concepts/)*
+
+#### Example projects
+
+*See also [Examples](/docs/examples/)*
+
+## Deep-dive: IT technologies and techniques
```

Comment:
I really like the content but I feel like **Introduction into IT/OT** and **Deep-dive: IT technologies and techniques** don't really belong logically together. I would maybe either split it up into two files under the **Getting Started** Category or I would move the content of **Deep-dive: IT technologies and techniques** into the **Tutorial** Category.

Reply:
We're my thoughts as well. I'll do it tomorrow.